var/spack/repos/builtin/packages/openmpi/package.py

```diff
@@ -40,6 +40,8 @@ class Openmpi(AutotoolsPackage, CudaPackage):
 
     version("main", branch="main", submodules=True)
 
+    version("5.0.0rc9", branch="main", tag="5.0.0rc9", submodules=True)
+
     # Current
     version(
         "4.1.4", sha256="92912e175fd1234368c8730c03f4996fe5942e7479bb1d10059405e7f2b3930d"
```

Comment:
You may want to add `preferred=True` to version 4.1.4 since it is the last stable release.

Reply:
Done